src/Extension/ExtensionTrait.php

```diff
@@ -26,7 +26,7 @@
 
     protected $extensionHidden = [
         'fields' => [],
-        'methods' => ['extensionIsHiddenField', 'extensionIsHiddenField']
+        'methods' => ['extensionIsHiddenMethod', 'extensionIsHiddenField']
```

Comment:
This is extremely pedantic I know but I would prefer that the order of these two are reversed because that's the order they appear in this file. :)